packages/x-date-pickers/src/locales/utils/pickersLocaleTextApi.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Set the types of the texts in the grid.
+ */
+export interface PickersLocaleText {
+  previousMonth: string;
+  nextMonth: string;
+  openPreviousView: string;
+  openNextView: string;
+  cancel: string;
+  clear: string;
+  ok: string;
+  today: string;
```

Comment:
Should we be more precise with the labels ?

```suggestion
  todayButtonLabel: string;
```